src/pfe/portal/routes/projects/links.route.js

```diff
@@ -165,9 +165,14 @@ async function getProjectURL(project) {
 
 async function restartProjectToPickupLinks(user, project) {
   if (global.codewind.RUNNING_IN_K8S) {
-    // In K8s we use configmaps and restart Pods
-    await updateNetworkConfigMap(project);
-    await restartDeployment(project);
+    const { extension } = project;
+    if (extension) {
```

Comment:
How will this work with other extension types? (Like ODO!)

Reply:
Right... Odo, I'll need to check this